src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -38,7 +38,7 @@ col-sm-2
 {%- endblock form_row %}
 
 {% block submit_row -%}
-    <div class="form-group">{#--#}
+    <div {% with {attr: row_attr.class is defined ? row_attr : row_attr|default({})|merge({class: 'form-group'})} %}{{ block('attributes') }}{% endwith %}>{#--#}
```

Comment:
`|default({})` is useless. `row_attr` is defined and is always an array, thanks to the FormType supporting it.

Reply:
Where because I can't find it? The default is already there in the form_div_layout.

Reply:
will be fixed in #33682

Reply:
Unresolving because that means I must remove all `|default` here.